api/v1alpha1/clusterconfig_types.go

```diff
@@ -40,6 +40,8 @@ type ClusterConfigSpec struct {
 	AWS *AWSSpec `json:"aws,omitempty"`
 	// +optional
 	Docker *DockerSpec `json:"docker,omitempty"`
+	// +optional
+	Nutanix *NutanixSpec `json:"nutanix,omitempty"`
```

Comment:
To make the reviewing easier what do you think about opening a PR with just API changes for only the required input?